<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectionLightSourceComponentView</class>
 <widget class="QWidget" name="DirectionLightSourceComponentView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DirectionLightSourceComponentView</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="intensitySpinBox">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="colorFrame">
     <property name="minimumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>35</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Color:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="colorButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Intensity:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Cast shadows:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="shadowComboBox">
     <item>
      <property name="text">
       <string>No shadows</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hard shadows</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
